src/Components/ImagesTable/ImageBuildErrorDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const useGetErrorReason = (err) => {
+    if (!err || !err.reason) {
+        return 'An unknown error occured';
+    }
+
+    if (err.details && err.details.reason) {
+        return err.details.reason;
+    }
+
+    return err.reason;
+};
+
+const FailureIcon = () => (
+    <Flex className="pf-u-align-items-baseline pf-m-nowrap">
+        <div className="pf-u-mr-sm">
+            <ExclamationCircleIcon className="error" />
+        </div>
+        <strong className='pf-u-danger-color-100'>Image build failed</strong>
+    </Flex>
+);
+
+const ErrorDetails = ({ status }) => {
+    if (!status || status.status !== 'failure') {
+        return <React.Fragment />;
```

Comment:
Would the short syntax for fragments `</>` work here? We use both throughout the code base but </>, I'm not sure whether we prefer one over the other... The short syntax is a newer feature...

Reply:
So both work. We should ideally only use one. I'm more in favor of the short `<> </>`

Reply:
I prefer `<></>`, I just saw that one of the other components was using `<React.Fragment>` so went with that for consistency. I'll update.